core/src/smartcontracts/isi/asset.rs

```diff
@@ -74,7 +74,7 @@ pub mod isi {
                     .map_err(|e| Error::Conversion(e.to_string()))?;
                 *quantity = quantity
                     .checked_add(self.object)
-                    .ok_or(Error::Math(MathError::Overflow))?;
+                    .ok_or(MathError::Overflow)?;
```

Comment:
While this is less explicit, the `?` does the `Into` conversion anyway.

Reply:
Yeah, since `#[from] MathError` implementation is seen in `Self::Error` declaration, I think this is enough readable

Reply:
Yeah, since `#[from] MathError` implementation is seen in `Self::Error` declaration, I think this is enough readable